pkg/handler/data_recorder_pubsub.go

```diff
@@ -61,9 +61,11 @@ func (p *pubsubRecorder) AsyncRecord(r *models.EvalResult) {
 		logrus.WithField("pubsub_error", err).Error("error marshaling")
 	}
 
-	ctx := context.WithTimeout(context.Background(), 10*time.Second) // TODO: make it configurable
+	ctx := context.Background()
 	res := p.topic.Publish(ctx, &pubsub.Message{Data: payload})
 	if config.Config.RecorderPubsubVerbose {
+		ctx, cancel := context.WithTimeout(ctx, 5*time.Second)
```

Comment:
What do you think about combining this with running the Get within a new goroutine?

Reply:
+1 for another goroutine. Also, 5s can be moved into env config.

Reply:
Agreed, I added both